cli/flags.rs

```diff
@@ -223,6 +226,18 @@ Examples: https://github.com/WICG/import-maps#the-import-map",
           }
         })
         .global(true),
+    ).arg(
+      Arg::with_name("lock")
+        .long("lock")
+        .value_name("FILE")
+        .help("Check the specified lock file")
+        .takes_value(true)
+        .global(true),
+    ).arg(
+      Arg::with_name("lock-write")
```

Comment:
I think it may be more user friendly to do `deno --lock-write lockfile.json` rather than `deno --lock lockfile.json --lock-write`.
I'd be okay with doing this in in a follow-up PR tho.